### ISSUE TITLE
Powershell addition would be *moi* Chefs kiss

### ISSUE THREAD

**Issue by matbeedotcom** [vallops99/Conda-autoactivate-env#4]

**Comment by vallops99** [comment]
Hey @matbee-eth 
That makes sense, I currently have no experience with Powershell, if you have some please feel to open a PR!

**Comment by vallops99** [comment]
Hey @matbee-eth 
A kind contributor (@cvanelteren ) made a nushell implementation of the script.
As far as I know nushell works with Windows.
In case you still need auto activation go checkout the `conda_switch.nu` file.

**Comment by cvanelteren** [comment]
Not quite; there is a minor bug that needs to be merged first see https://github.com/mamba-org/mamba/pull/3771
